_posts/2016-12-30-Is-not-a-Mass-is-just-a-stand-up.markdown

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title:  "Is not a Mass, is just a stand-up"
+date:   2016-12-30 11:28:00 -0300
+categories: software, stand-up, scrum
+---
+
+Each day, at 12.00 am, the priest is ready to start the mass. It doesn’t really matter what you did or not the last day or even the quality of the code, the only most important thing is that you be there on the stand-up and talk about something.
```

Comment:
The word priest is very catholic, may be pastor, lord or minister could work better.